saltychat/SaltyClient/External/Tasks.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CitizenFX.Core;
+using CitizenFX.Core.Native;
+using System;
+using System.Threading.Tasks;
+
+namespace RedM.External
+{
+    public class Tasks
+    {
+        private Ped Ped { get; }
+
+        internal Tasks(Ped ped)
+        {
+            Ped = ped;
+        }
+
+        /// <summary>
+        /// Custom addition for facial animation native calls
+        /// </summary>
+        /// <param name="animDict"></param>
+        /// <param name="animName"></param>
+        public async Task PlayFacialAnimation(string animDict, string animName)
+        {
+            if (!Function.Call<bool>(Hash.HAS_ANIM_DICT_LOADED, animDict))
+            {
+                Function.Call(Hash.REQUEST_ANIM_DICT, animDict);
+            }
+
+            var end = DateTime.UtcNow.AddMilliseconds(1000f);
```

Comment:
I'd personally stick with the game timer (ms since start) instead of working with `DateTime`
```suggestion
            int end = API.GetGameTimer() + 1000;
```